test/prototype_transforms_kernel_infos.py

```diff
@@ -854,7 +854,7 @@ def sample_inputs_rotate_video():
             F.rotate_bounding_box,
             sample_inputs_fn=sample_inputs_rotate_bounding_box,
             closeness_kwargs={
-                **scripted_vs_eager_double_pixel_difference("cpu", atol=1e-6, rtol=1e-6),
```

Comment:
Is this a driveby?

Reply:
Yes, 

> Increase tolerence for flaky cpu-bbox-rotation eager vs scripted tests

as they can fail in a flaky way:
- https://github.com/pytorch/vision/actions/runs/4164786358/jobs/7206896467
- https://github.com/pytorch/vision/actions/runs/4164786358/jobs/7206895696